app/models/external_registry/verloren_of_gevonden_bike.rb

```diff
@@ -0,0 +1,91 @@
+module ExternalRegistry
+  class VerlorenOfGevondenBike < ExternalRegistryBike
+    def external_registry_name
+      "verlorenofgevonden.nl"
+    end
+
+    def external_registry_url
+      "https://verlorenofgevonden.nl"
+    end
+
+    def url
+      [external_registry_url, "overzicht?search=#{external_id}"].join("/")
+    end
+
+    def image_url
+      return if info_hash["object_id"].blank?
+      [external_registry_url, "assets", "image", info_hash["object_id"]].join("/")
+    end
+
+    def thumb_url
+      image_url
+    end
+
+    class << self
+      def build_from_api_response(attrs = {})
+        is_bike = attrs["Category"] == "fiets"
+        return unless is_bike
+
+        description = attrs["Description"].presence
+
+        bike = find_or_initialize_by(
+          external_id: attrs["ObjectNumber"].presence,
+          type: "ExternalRegistry::VerlorenOfGevondenBike",
```

Comment:
I don't like this naming pattern because it doesn't seem accurate. I'd prefer it if it were `ExternalRegistryBike:: VerlorenOfGevondenBike` - since that what it's actually inheriting from. I'm happy with two different folders.

Reply:
Updated in 3526cb39 -- went with the pluralized form since the singular conflicts with the ActiveRecord class.